src/robot_interface/telemetry/payloads.py

```diff
@@ -36,8 +36,8 @@ class RobotStatusPayload:
     robot_status: RobotStatus
     current_isar_state: State
     current_mission_id: Union[UUID, int, str, None]
-    current_task_id: UUID
-    current_step_id: UUID
+    current_task_id: Union[UUID, str]
+    current_step_id: Union[UUID, str]
```

Comment:
Should we change to only use str? I think Union makes for confusion and I dont see the point. Imo we just change the UUID to a str as soon as it is created, then keep it str

Reply:
Agree